symphony/lib/boot/func.utilities.php

```diff
@@ -8,32 +8,32 @@
  * Redirects the browser to a specified location. Safer than using a
  * direct header() call
  *
- *  @param string $url
+ *  @param string $location
+ *  @param string $status (optional)
  */
-function redirect ($url)
+function redirect($location, $status = '302 Found')
 {
-    // Just make sure.
-    $url = str_replace('Location:', null, $url);
-
     if (headers_sent($filename, $line)) {
-        echo "<h1>Error: Cannot redirect to <a href=\"$url\">$url</a></h1><p>Output has already started in $filename on line $line</p>";
-        exit;
-    }
 
-    // convert idn back to ascii for redirect
+        // throw exception if headers already sent
```

Comment:
Fair enough. I find it the opposite. Splitting a 4 line function over multiple lines with the extra whitespace messes with the grouping in my head. For example I find the top more readable then the bottom, especially when scanning through code quickly (either by scrolling or with Sublime's preview)

```php
// Comment
if (condition) {
 // do condition
}
```

Instead of

```php
// Comment

if (condition)

// do comment

}
```